Scripts/emoji_ranges.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+import sys
+import os
+import re
+import commands
+import subprocess
+import io
+
+def fail(message):
+    print message
+    sys.exit(1)
+
+big_ranges = [
+        (0x1F600,0x1F64F, ),
+        (0x1F300,0x1F5FF, ),
+        (0x1F680,0x1F6FF, ),
+        (0x2600,0x26FF,   ),
+        (0x2700,0x27BF,   ),
+        (0xFE00,0xFE0F,   ),
+        (0x1F900,0x1F9FF,  ),
+        (65024,65039,), 
+        (8400, 8447,)
+        ]
+
+# def is_in_big_range(range_start, range_end)
+
+if __name__ == '__main__':
```

Comment:
This script converts the emoji ranges from the unicode spec to more usable ranges by combining adjacent ranges.